Frameworks/AddressBook/_ABAddressBook.mm

```diff
@@ -0,0 +1,52 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "_ABAddressBook.h"
+#import "UWP/WindowsApplicationModelContacts.h"
+
+@interface _ABAddressBook ()
+
+@end
+
+@implementation _ABAddressBook
```

Comment:
>_ABAddressBook [](start = 16, length = 14)

I'm not sure you need an extra class to handle the fetch for the contact store. It might be better to break it out into a static helper function and store the contact store as a private ivar in address book.